lib/webdriver.py

```diff
@@ -144,14 +144,18 @@ def _get_driver(self) -> Driver:
             driver_version=driver_version,
             headless=True,
             uc_cdp_events=True,
-            undetectable=True,
+            undetectable=True
         )
         logger.debug("Using browser version: %s", driver.caps["browserVersion"])
 
         driver.add_cdp_listener("Network.requestWillBeSent", self._headers_listener)
 
         logger.debug("Loading Southwest Check-In page")
-        driver.get(CHECKIN_URL)
+        driver.open(CHECKIN_URL)
+        driver.type("input[name='recordLocator']", RECORD_LOCATOR)
+        driver.type("input[name='firstName']", "fname")
+        driver.type("input[name='lastName']", "lname")
```

Comment:
There may want to be logic here for using the real first/last name of an account. Not sure how big of a deal it would be to have the same reservation info every single time (although this is what the script used to do, so it's probably fine).